src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/query.js

```diff
@@ -7,18 +7,28 @@
  */
 
 import { offsetTime } from '../../offset_time';
-import { getIntervalAndTimefield } from '../../get_interval_and_timefield';
 import { esQuery } from '../../../../../../data/server';
 
-export function query(req, panel, series, esQueryConfig, seriesIndex) {
-  return (next) => (doc) => {
-    const { timeField } = getIntervalAndTimefield(panel, series, seriesIndex);
+export function query(
+  req,
+  panel,
+  series,
+  esQueryConfig,
+  seriesIndex,
+  capabilities,
```

Comment:
Why do we add   capabilities and uiSettings? They are not used anywhere in the query function.

Reply:
Yes, this is a disgusting part of processor implementations. This is a function that accepts a sequence of arguments. So it's important for us to keep order

Reply:
Do you want to add a comment? In order to not confuse us  :D

Reply:
It happens across all request/response processors. I'll create an issue to fix that